### PR TITLE
Add AdSpeed ad server to amp-ad

### PR DESCRIPTION
**Pull request by Adspeed**
Hello,

I've followed the guidelines and examples to add support for AdSpeed ad server into amp-ad. 

- No error with gulp lint
- Ran & passed tests "test-ads-config.js,test-integration.js"
- Tested locally with "examples/ads.amp.max.html?type=adspeed"
- Signed Google CLA (git username: "adspeed")

Please review and let me know. Thanks!

### CODE REVIEW

File: ads/_config.js [hunk 1]
```diff
@@ -84,6 +84,10 @@ export const adConfig = {
     preconnect: 'https://api.adsnative.com',
   },
 
+  adspeed: {
+    preconnect: 'https://g.adspeed.net',
+  },
```
Comment (lannka) [inline ads/_config.js]: could you also implement the render-start and no-content API?

**Comment by Adspeed** [comment]
Hello, I've just updated our backend JS code to call the API renderStart() and noContentAvailable() when suitable and I modified ads/_config.js. However, Travis CI build failed (error below), which I don't think is because of our code, looks like a config thing. Please check and restart build if it's the case. Thanks

"The command "node build-system/pr-check.js" exited with 1."

**Comment by Adspeed** [comment]
Hello. Can someone help restart Travis build? Thanks!

**Comment by jridgewell** [comment]
Restarted.

**Comment by Adspeed** [comment]
It failed very early again at: "pr-check.js - any update to package.json or yarn.lock must include the other file. Please update through yarn."

Anyone knows why? 

I re-ran these local checks on my machine without any issue:
- gulp lint
- gulp check-types
- gulp build
- gulp serve

Thanks for your help

**Comment by Adspeed** [comment]
Hello,

Any help with the build config/troubleshooting/result? 

Or should I close this PR and restart using the latest amphtml revision?

Thanks!

**Comment by jridgewell** [comment]
Try rebasing against current master? Assuming you called [`ampproject/amphtml`](github.com/ampproject/amphtml) the `upstream` remote:

```bash
$ git pull --rebase upstream/master
```

**Comment by Adspeed** [comment]
Thanks Justin! That (updating/syncing the fork) definitely helped and all checks have passed now. Please review.

**Comment by Adspeed** [comment]
Thanks!!

**Pull request merged**